### PR TITLE
#20 Run ktlint on pr create

### LINKED ISSUE #20: Run ktlint on pr create
To keep one code convention it will be nice to run ktlint check on every pr.

Check if it is possible to write custom ktlint rules using ktlint Gradle plugin. If it is not possible than go with the Custom Gradle integration.

Put the results of the findings in the comment.

https://pinterest.github.io/ktlint/install/integrations/#gradle-integration 

- [x] Add ktlint
- [x] Format code
- [x] Run ktlint on pr create

### PR DESCRIPTION
**Pull request by bartek977**

### CODE REVIEW

**Review by michal-kucznerowicz — APPROVED** [review]

File: .github/workflows/default_branch_verification.yml [hunk 1]
```diff
@@ -20,5 +20,7 @@ jobs:
 
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      - name: Analyze
+        run: ./gradlew ktlintCheck
```
Comment (Nataniel-Antosik) [inline .github/workflows/default_branch_verification.yml]: Could you look at the logs from action for example for this PR 
image
Does this indicate we have already `ktlint` in the `build`? If we will don't use the `build` for GitHub action and w use `assemble` then we should consider writing jobs for `ktlint`, `lint`, `tests`

Comment (Nataniel-Antosik) [reply .github/workflows/default_branch_verification.yml]: This logs was added when you create PR #32

File: .github/workflows/pull_request_verification.yml [hunk 2]
```diff
@@ -20,5 +20,7 @@ jobs:
 
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      - name: Analyze
+        run: ./gradlew ktlintCheck
```
Comment (Nataniel-Antosik) [inline .github/workflows/pull_request_verification.yml:24]: Same situation as here: https://github.com/intive/Picover/pull/34/files#r1106782207

**Comment by Nataniel-Antosik** [comment]
Minor: You named PR to add ktlint for creating PR and you add checking ktlint for Pull requests and also for branch develop.

**Comment by bartek977** [comment]
> Minor: You named PR to add ktlint for creating PR and you add checking ktlint for Pull requests and also for branch develop.

fixed

File: .github/workflows/default_branch_verification.yml (hunk 1 above)
Comment (bartek977) [reply .github/workflows/default_branch_verification.yml]: These logs are about building our `ktlint-ruleset` module where we will put custom rules

**Review by Nataniel-Antosik — APPROVED** [review]

**Pull request merged**